lib/beaker/host/windows/file.rb

```diff
@@ -9,6 +9,32 @@ def tmpdir(name)
     execute("cygpath -m $(mktemp -td #{name}.XXXXXX)")
   end
 
+  def system_temp_path
+    # under CYGWIN %TEMP% may not be set
+    tmp_path = execute('ECHO %SYSTEMROOT%', :cmdexe => true)
+    tmp_path.gsub(/\n/, '') + '\\TEMP'
+  end
+
+  # Updates a file path for use with SCP, depending on the SSH Server
+  #
+  # @note This will fail with an SSH server that is not OpenSSL or BitVise.
+  #
+  # @param [String] path Path to be changed
+  #
+  # @return [String] Path updated for use by SCP
+  def scp_path(path)
+    if is_ssh_server_bitvise?
```

Comment:
Same comment here about the API... why use the extra `is_xxx_` style methods, when the alternative is simpler / more resilient to change?

```ruby
case ssh_server_type
when :openssl
  path
when :bitvise
  network_path = path.gsub('\\', scp_separator)
  network_path.gsub('C:', '')
else
  raise ArgumentError
end
```